src/igtcloud/client/tools/download_institute.py

```diff
@@ -41,11 +42,15 @@ def download_institutes(project_name: str, institute_name: str, destination: str
         if not categories:
             categories = ['files']
 
-        with concurrent.futures.ThreadPoolExecutor(max_workers=4) as executor:
+        with concurrent.futures.ThreadPoolExecutor(max_workers=max_workers_studies or 4) as executor:
```

Comment:
`ThreadPoolExecutor` by default sets `max_workers` to `min(32, os.cpu_count() + 4)`. Perhaps that's a more efficient default value to set than `4`.

Reply:
Each job submitted to this pool creates a new pool of again 4 threads. This results in (by default) 16 threads for parallel downloads.
This is usually sufficient to consume all available bandwidth.